### ISSUE TITLE
Quality control of Electives work

### ISSUE THREAD

**Issue by docsteveharris** [HYLODE/HyUi#301]

**Comment by docsteveharris** [comment]
@hqva and @JenHunter89 are we blocked on this b/c we can't debug HyUi properly?

**Comment by hqva** [comment]
Yes - haven't yet got HyUI working on gae08 to properly troubleshoot / fix the date selector to then start properly cross-checking.
